static/bidder-params/bliink.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Bliink Adapter Params",
+  "description": "A schema which validates params accepted by the Bliink adapter",
+  "type": "object",
+
+  "properties": {}
+}
```

Comment:
Just to confirm - you expect no bidder parameters at all?

Reply:
While testing it locally without any bidder parameters I get empty response. Is it expected?

Reply:
Hi, yes, we don't expect bidder parameters.